article-analysis-project/collect_articles.py

```diff
@@ -0,0 +1,26 @@
+import requests, mediacloud.api
+
+def article_urls(id):
+    start = 0
+    rows  = 1000
+    while True:
+        params = {
+            'last_processed_stories_id': start,
+            'rows': rows,
+            'q': 'media_id:' + str(id),
+            'fq': 'publish_date:[2020-06-29T07:00:00Z TO 2020-06-30T09:00:00Z]',
+            'key': '56db1b3371386348692be15924705bf1c3530d7ef82c97e327073768d88f6aa1'
+        }
+
+        #print("Fetching {} stories starting from {}").format( rows, start)
+        r = requests.get( 'https://api.mediacloud.org/api/v2/stories_public/list/', params = params, headers = { 'Accept': 'application/json'} )
```

Comment:
This might sound crazy but I would recommend putting the media cloud API root in a constant, and then appending the endpoint that you want to access.